Add remote bfshell support (#585)
stratum/stratum: stratum/hal/bin/barefoot/README.run.md

need to press "Enter" a few times to see the prompt. Please refer to BF SDE
documentation for details on how to use the CLI.

### Access BF Shell via telnet (experimental)

> When Stratum started, the BF Shell will be enbaled by default.

Reviewer: ```suggestion
When Stratum started, the BF Shell will be enabled by default.
```

To access the BF Shell, use the `telnet` command:

```bash
telnet [Switch IP] 9999
```

> Once enter the BF Shell, type `ucli` to access the BF CLI.

Reviewer: ```suggestion
Once you have entered the BF Shell, type `ucli` to access the BF CLI.
```

> > Since we are not support all features from the BF SHell, using commands other than `ucli` may cause Stratum crash.

Reviewer: ```suggestion
> Since we do not support all features from the BF Shell, using commands other than `ucli` may cause Stratum crash.
```

> To exit the telnet, press `Ctrl` and `]` to close the connection and type `quit` to leave the telnet.

Reviewer: ```suggestion
To exit the telnet session, press `Ctrl` and `]` to close the connection and type `quit` to leave the telnet.
```
